src/components/views/rooms/MessageComposer.js

```diff
@@ -325,16 +337,28 @@ export default class MessageComposer extends React.Component {
                      key="controls_formatting" />
             );
 
-            const placeholderText = roomIsEncrypted ?
-                _t('Send an encrypted message') + '…' : _t('Send a message (unencrypted)') + '…';
+            let placeholderText;
+            if (this.state.isQuoting) {
+                if (roomIsEncrypted) {
+                    placeholderText = _t('Send an encrypted reply');
+                } else {
+                    placeholderText = _t('Send a reply (unencrypted)');
+                }
+            } else {
+                if (roomIsEncrypted) {
+                    placeholderText = _t('Send an encrypted message');
+                } else {
+                    placeholderText = _t('Send a message (unencrypted)');
+                }
+            }
 
             controls.push(
                 <MessageComposerInput
                     ref={(c) => this.messageComposerInput = c}
                     key="controls_input"
                     onResize={this.props.onResize}
                     room={this.props.room}
-                    placeholder={placeholderText}
+                    placeholder={placeholderText + '…'}
```

Comment:
We should probably put this in the literal strings if we're adding more: I'm sure this doesn't work in all languages.